README.md

```diff
@@ -19,50 +17,67 @@ A few Sphinx extensions are used as well:
   in the API docs and tutorials. This is also modified from its original state, and that
   fork can be found at [this fork](https://github.com/purduesigbots/sphinx-tabs).
 
-## Contributing
+## I Just Want to Help
+Great! Please help us. Choose one of the following that best suits your needs. 
+
+#### Easy

```

Comment:
For the easy/medium/hard distinction I just came up with something that sounded good. I'm not sold on that *wording* per say, but I do like having different methods that people can use.

Reply:
I'm not entirely sure what you mean by having the sections. Would you like to elaborate on that a little?

Reply:
more like what was already in what is now the "hard" section-- section titles that describe a certain kind of change as opposed to how hard that change may be to make

Reply:
I like that idea. I just commited a version of the README that rather than: easy, medium, and hard, they are Simple Text Changes, Formatting Changes, and Theme Changes.